content/Video_Display.md

```diff
@@ -290,46 +290,46 @@ of this brightness correction.
 
 Writing to this register launches a DMA transfer from ROM or RAM to OAM
 memory (sprite attribute table). The written value specifies the
-transfer source address divided by 100h, that is, source & destination are:
+transfer source address divided by $100, that is, source and destination are:
 
 ```
-Source:      XX00-XX9F   ;XX in range from 00-F1h
-Destination: FE00-FE9F
+Source:      $XX00-$XX9F   ;XX = $00 to $F1
+Destination: $FE00-$FE9F
 ```
 
 The transfer takes 160 machine cycles: 152 microseconds in normal speed
 or 76 microseconds in CGB Double Speed Mode. On DMG, during this time,
-the CPU can access only HRAM (memory at FF80-FFFE); on CGB, the bus used
+the CPU can access only HRAM (memory at $FF80-$FFFE); on CGB, the bus used
 by the source area cannot be used (this isn't understood well at the
-moment, it's recommended to assume same behavior as DMG). For this
+moment; it's recommended to assume same behavior as DMG). For this
 reason, the programmer must copy a short procedure into HRAM, and use
 this procedure to start the transfer from inside HRAM, and wait until
 the transfer has finished:
 
 ```
  run_dma:
-  ld a, start address / 100h
-  ldh  (FF46h),a ;start DMA transfer (starts right after instruction)
-  ld  a,28h      ;delay...
+  ld a, start address / $100
+  ldh  ($FF46),a ;start DMA transfer (starts right after instruction)
```

Comment:
RGBDS syntax uses brackets instead of parens.

Reply:
Fixed